### ISSUE TITLE
key-duplicates across files

### ISSUE THREAD

**Issue by patrickberkeley** [adrienverge/yamllint#225]
Is it possible to check for duplicate keys across files? E.g.,

Run: `yamllint **/*.yml`

File 1:

```yml
key_1: 'val'
```
File 2 has the same key:

```yml
key_1: 'val'
```

The linter would fail in this case.

**Comment by adrienverge** [comment]
Hello,

No it's not possible to do this with yamllint.

What you ask is a bit uncommon, do you want to describe your use-case?

**Comment by patrickberkeley** [comment]
Thanks for the quick response @adrienverge.

Yes we have a directory of yml files used for translations. They are separated for clarity, but occasionally we've run into issues where someone will unwittingly clobber an existing top-level key. It would be nice if we could catch that in our linting.

**Comment by adrienverge** [comment]
Although it's not possible directly with yamllint, I would recommend concaneting these files and tell yamllint to read from standard input. For example:

`a.yaml`:
```yaml
key: val
foo: bar
```
`b.yaml`:
```yaml
foo: baz
```

`cat *.yaml | yamllint -`
```
stdin
  1:1       warning  missing document start "---"  (document-start)
  3:1       error    duplication of key "foo" in mapping  (key-duplicates)
```

**Comment by patrickberkeley** [comment]
Awesome, thanks @adrienverge
